src/threadpool.c

```diff
@@ -33,12 +33,16 @@ static uv_once_t once = UV_ONCE_INIT;
 static uv_cond_t cond;
 static uv_mutex_t mutex;
 static unsigned int idle_threads;
+static unsigned int slow_io_work_running;
 static unsigned int nthreads;
 static uv_thread_t* threads;
 static uv_thread_t default_threads[4];
 static QUEUE exit_message;
 static QUEUE wq;
+static QUEUE run_slow_work_message;
+static QUEUE slow_io_pending_wq;
 
+#define SLOW_WORK_THREAD_THRESHOLD ((nthreads + 1) / 2)
```

Comment:
Maybe make this a function.  The fact that it's a macro hides that it's not a constant but depends on a global.

Reply:
Done!